python/cudf/cudf/tests/test_parquet.py

```diff
@@ -901,6 +902,57 @@ def test_parquet_reader_list_large_mixed(tmpdir):
     assert_eq(expect, got, check_dtype=False)
 
 
+def test_parquet_reader_list_large_multi_rowgroup(tmpdir):
+    # > 800 row groups
+    num_rows = 2_000_000
```

Comment:
Can this test be smaller? 2MM rows and 800 row groups sounds like a non-trivial runtime.

Reply:
Sure.  It runs pretty quick (5 seconds? locally).  I'll drop by half.

Reply:
> It runs pretty quick (5 seconds? locally)

That would easily make it in the 1% of slowest tests. We have close to 75k unit tests nowadays where if they all ran even a second it would be painful. Does this not reproduce with something even more minimal?

Reply:
Definitely.  Reproduces with just 2 row groups.  I just wanted to make sure it exercised as much as possible.  I'll drop even further.